graylog2-web-interface/src/components/sidecars/sidecars/VerboseMessageModal.jsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Button, Modal } from 'react-bootstrap';
+import lodash from 'lodash';
+
+import BootstrapModalWrapper from 'components/bootstrap/BootstrapModalWrapper';
+import CombinedProvider from 'injection/CombinedProvider';
+
+const { CollectorConfigurationsActions } = CombinedProvider.get('CollectorConfigurations');
+
+class VerboseMessageModal extends React.Component {
+
+  open = () => {
+    this.sourceModal.open();
+  };
+
+  hide = () => {
+    this.sourceModal.close();
+  };
+
+  render() {
+    return (
+      <BootstrapModalWrapper ref={(c) => { this.sourceModal = c; }} bsSize="large">
+        <Modal.Header closeButton>
+          <Modal.Title><span>Error Details for <em>{this.props.collectorName}</em></span></Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <div className="configuration">
+            <pre>
+              {this.props.collectorVerbose || '<no messages>'}
```

Comment:
This prop should be included in a `propTypes` declaration (https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md).